codespell_lib/data/dictionary.txt

```diff
@@ -20463,6 +20463,7 @@ occurrance->occurrence
 occurrances->occurrences
 occurrencs->occurrences
 occurrs->occurs
+oce->one
```

Comment:
It should definitely suggest `once`, probably first and possibly only (@matkoniecz that may also be the correction you should actually be using too). Otherwise it should suggest `ore` too, which is at least the same side of the keyboard.

Reply:
In my case it was definitely `one` (I do not remember exact commit or project, but it was definitely referring to number).